### PR TITLE
Docs minor updates to letters, self-mailers

### PR DESCRIPTION
**Pull request by maelingmurphy**
Fixes #\<INSERT ISSUE NUMBER HERE\>

## Checklist

- [X] Up to date with `main`
- [X] All the tests are passing (All except
  - [ ] Delete all resources created in tests
- [X] Prettier
- [X] Spectral Lint
- [X] `npm run bundle` outputs nothing suspect
- [x] `npm run postman` outputs nothing suspect

## Changes
- Update `sort_by` shared parameter description to include generic `items` text instead of `postcards` since it is being used across different resources
- Add `11x9_bifold` enum to `self_mailer_size.yml`
- ~Add `cards` property to `letters` resource file - `no_extra_service.yml`~
- [ ]  Figure out why `cards` property is not displaying for `letters` LIST endpoint in docs
-  Rebuild redoc index.html 

## Areas of Concern (optional)
- Related to enums, there's a couple of other minor fields that we might make as enums. Like orientation & object in the `cards` resource

### CODE REVIEW

File: dist/lob-api-bundled.yml [hunk 1]
```diff
@@ -3105,7 +3105,7 @@ components:
       in: query
       name: sort_by
       description: |
-        Sorts postcards by ascending or descending dates.
+        Sorts items by ascending or descending dates.
```
Comment (Kilo59) [inline dist/lob-api-bundled.yml:3108]: 👍

**Review by Kilo59 — APPROVED** [review]

File: dist/lob-api-bundled.yml (hunk 1 above)
Comment (Kilo59) [reply dist/lob-api-bundled.yml:3108]: Alternate noun 🤷 
>Sorts collection by ...

But I like `items`.

**Comment by Kilo59** [comment]
Talking to @maelingmurphy and we think we need to re-build the redoc `docs/index.html` so that `letters` has the new cards examples.
That's coming shortly.

**Comment by sudoku-lord** [comment]
Since this has a conflict, I'm closing it and creating a new PR with the desired changes rolled in.

**Pull request closed without merging**